src/ops.jl

```diff
@@ -219,8 +217,8 @@ function to_function(op::tensorflow.OpDef)
     input_block = quote end
     convert_block = quote end
     type_sets = Dict{String, Vector{Symbol}}()
-    for input in op.input_arg
-        sym = gensym()
+    for (i, input) in enumerate(op.input_arg)
+        sym = Symbol(:x, i)
```

Comment:
I feel like this can be `Symbol(op.input_arg.name)`.
Which will match the docs.
But possibly that would make a name clash?
If so appending a underscore or similar might be the trick.